packages/libs/map/types.ts

```diff
@@ -102,6 +102,18 @@ export type Ferry = Readonly<{
   connections: FerryConnection[];
 }>;
 
+export type MileageTarget = Readonly<{
+  token: string;
+  editorName: string;
+  defaultName: string;
+  nameVariants: string[];
+  distanceOffset: number;
+  nodeUid?: string;
+  easting?: number;
+  southing?: number;
```

Comment:
The usual axes of a spatial cartesian coordinate system are *easting-northing-up*. Since SCS's choice of *easting-up-southing* is quite uncommon, using clear names for coordinate properties and variables is particularly important. In my view, `x` and `y` don't live up to that.

Even so, consistency is a winning argument, I suppose.

Renaming, and adding a few comments.